norse/benchmark/main.py

```diff
@@ -58,7 +58,9 @@ def benchmark(
                 logging.error(message)
 
             data = BenchmarkData(
-                config=config, durations=np.array(durations), parameters=parameters,
+                config=config,
```

Comment:
How did this whitespace change happen?

Reply:
Black got upgraded. I just added the new formatting because it's in our C